tests/Integration/Console/Scheduling/ScheduleListCommandTest.php

```diff
@@ -120,7 +120,7 @@ public function testDisplayScheduleSubMinute()
 
         $this->artisan(ScheduleListCommand::class)
             ->assertSuccessful()
-            ->expectsOutput('  * 0 * * 0 1s   php artisan inspire ............. Next Due: 1 second from now')
+            ->expectsOutput('  * 0 * * 0 1s   php artisan inspire ............... Next Due: 6 days from now')
```

Comment:
Please undo this and rebase with 10.x, thanks.

Reply:
Okay